manifests/Postman/Postman/7.24.0.yaml

```diff
@@ -10,7 +10,7 @@ Homepage: https://www.postman.com/
 Installers:
   - Arch: x64
     Url: https://dl.pstmn.io/download/latest/win64?filename=Postman-win64-7.24.0-Setup.exe
```

Comment:
Should be `https://dl.pstmn.io/download/version/7.24.0/win64` as @chausner noted.